src/style.css

```diff
@@ -1 +1,69 @@
 /* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(/src/fonts/Avenir.ttc);
+}
+
+.wrapper {
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+}
+
+.input-wrapper__big input {
+  width: 100%;
+  padding: 20px;
+  border: 1px solid #ccc;
+  border-radius: 5px;
+  padding-left: 50px;
+}
+
+.input-wrapper__small input {
+  width: 427px;
+  padding: 10px;
+  border: 1px solid #ccc;
+  border-radius: 5px;
+  padding-left: 30px;
+}
+
+.input-wrapper__big input:hover {
+  box-shadow: 0 3px 8px 0 #E1E7ED;
+  border-radius: 5px;
+}
+
+.input-wrapper__big input:focus {
+  outline: none;
+  box-shadow: 0 4px 4px 0 #E1E7ED;
+  font-size: 16px;
+  font-weight: 900;
+  line-height: 22px;
+  color: #3d4e61;
+}
+
+.input-with-icon-1 {
+  background-image: url(/src/images/Search.svg);
+  background-position: left 20px center;
+  background-repeat: no-repeat;
```

Comment:
background-image and background-repeat properties the same for both inputs. Consider to use 1 class to define this properties.